Python/Tests/GlassTests/PythonTests/RegisterPythonEngine.cmd

```diff
@@ -21,7 +21,7 @@ if "%RegistryRoot%"=="ERROR" goto IncorrectRegRootFormatError
 
 echo Importing PythonEngine.regdef to registry...
 if exist %tmp%\PythonEngine.reg del %tmp%\PythonEngine.reg>NUL
-call %~dp0GlassRegGen.exe %~dp0PythonEngine.regdef %RegistryRoot% %tmp%\PythonEngine.reg
+call C:\GlassTesting\GlassStandAlone\Glass\GlassRegGen.exe %~dp0PythonEngine.regdef %RegistryRoot% %tmp%\PythonEngine.reg
```

Comment:
doesn't RegisterPythonEngine.cmd need to be deleted? we no longer call it.

Reply:
Deleted